packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -395,6 +404,13 @@ export interface IContainerRuntimeOptions {
 	 * @experimental Not ready for use.
 	 */
 	readonly chunkSizeInBytes?: number;
+
+	/**
+	 * Enable the IdCompressor in the runtime.
+	 * @experimental Not ready for use.
+	 */
+	readonly enableRuntimeIdCompressor?: boolean;
```

Comment:
Is there any value to have here tri-state? Feels like boolean is sufficient. Is this only due to back-compat?